fileglancer/paths.py

```diff
@@ -89,6 +57,7 @@ def __init__(self, central_url: str, dev_mode: bool, jupyter_root_dir: str):
                             FileSharePath(
                                 zone=zone,
                                 name=name,
+                                owner=zone.lower().replace(' ', '_'),
```

Comment:
Wasn't this field removed recently? File share paths don't have owners...

Reply:
good catch - it should not be there